internal/evaluation_target/application_snapshot_image/application_snapshot_image.go

```diff
@@ -63,11 +64,13 @@ var attestationSchemas = map[string]jsonschema.Schema{
 
 // ApplicationSnapshotImage represents the structure needed to evaluate an Application Snapshot Image
 type ApplicationSnapshotImage struct {
-	reference    name.Reference
-	checkOpts    cosign.CheckOpts
-	signatures   []signature.EntitySignature
-	attestations []attestation.Attestation
-	Evaluators   []evaluator.Evaluator
+	reference        name.Reference
+	checkOpts        cosign.CheckOpts
+	signatures       []signature.EntitySignature
+	configJSON       json.RawMessage
+	parentConfigJSON json.RawMessage
```

Comment:
We never care about parent's parent? Could this be:
```suggestion
	configJSON       map[name.Reference]json.RawMessage
	path             map[name.Reference]name.Reference
```

Reply:
I think that makes it much harder to use the input from the policy.

> We never care about parent's parent?

Not that I know of. I can change `input.image.parent_config` to `input.image.parent.config`. That should allow us to do something like this in the future if needed: `input.image.parent.parent.parent.parent.config`

Reply:
This actually works out quite nice because I can now easily add `input.image.parent.ref` as well which could be useful to some policy rules.